src/main/java/use_case/getpost/GetPostInputData.java

```diff
@@ -0,0 +1,17 @@
+package use_case.getpost;
+
+/**
+ * The Input Data for the Get Post Use Case.
+ */
+public class GetPostInputData {
+
+    private final String entryID;
+
+    public GetPostInputData(String entryID) {
+        this.entryID = entryID;
+    }
+
+    public String getEntryID() {
```

Comment:
just a reminder, I'm working on post dao rn, so when I push it (hopefully before tuesday or wednesday) u may need to change the method names to match the methods in the dao. We can do this later in a seperate PR :)

Reply:
sounds good!